scripts/cbw_map_func.js

```diff
@@ -0,0 +1,141 @@
+/**
+ * Created by wsiddall on 09/03/2021
+ */
+let debug = false;
+let hostname = document.URL.split("/")[2].split(":")[0] === "localhost" ? "localhost:8080" : "ava-proto.com"
+let avadepthSFE = `https://${hostname}/avadepthserver/services/ows/wmts/avadepth`;
+let SFETileFormat = "default/{TileMatrixSet}/{TileMatrix}/{TileRow}/{TileCol}.png";
+
+let layerMap = {
+    channel: ["channel_cells", "channel_outline"],
+    soundings: ["soundings"]
+}
+
+avaMapJS.cbw_func = {
+    init: function() {
+        let format_cap = new OpenLayers.Format.WMTSCapabilities({
+            yx: {
+                "urn:ogc:def:crs:EPSG::3857": false
+            }
+        })
+        avaMapJS.cbw_func.wmts_layers = {}
+        OpenLayers.Request.GET({
+            url: avadepthSFE,
+            params: {
+                service: "WMTS",
+                version: "1.0.0",
+                request: "GetCapabilities"
+            },
+            success: function(request){
+                let doc = request.responseXML;
+                if (!doc || !doc.documentElement){
+                    doc = request.responseText;
+                }
+                let capabilities = format_cap.read(doc);
+                avaMapJS.cbw_func.wmts_layers = {
+                    combined: format_cap.createLayer(capabilities, {
+                        layer: "BathyAnalysis.Combined",
+                        name: "Combined",
+                        matrixSet: "GoogleMapsCompatible",
```

Comment:
We've had to switch over Bing maps as our maps provider because of the way Google changed their billing structure.  Will this cause any issues with using this GoogleMapsCompatible matrix set.

Reply:
Not at all. "GoogleMapsCompatible" is the identifier name in SFE's web service capabilities file, and it simply refers to the coordinate system being EPSG:3587.

Reply:
Excellent.  Looking really clean so far.  Will dig deeper in tomorrow.  Nice work.